KebPOS/UI/MainUI.cs

```diff
@@ -8,49 +8,23 @@ namespace KebPOS.UI
 {
     public class MainUI
     {
-        public MainUI()
-        {
-
-        }
-
         public void DisplayProductsTable(List<Product> products)
```

Comment:
You should be able to take a generic list as an argument and populate the columns based on the properties of the incoming type (you might need to use reflection). Let me know if you have questions.

Reply:
Should this just be a table that can be used dynamically with any form of data❓

Reply:
Ideally we want to use the UI class to show everything, including the main menu and prompts. But I think for now only the lists will be enough. What do you think?